plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -233,6 +234,16 @@ class NewCreateServiceModalForm extends Component {
     this.setState({batch, appConfig: this.getAppConfig(batch)});
   }
 
+  handleSetItem({value, path}) {
```

Comment:
Hm... could you please elaborate why do we need one more entry point for transactions?
Could we just keep an internal state in `MultiContainerHealthChecksFormSection.js` and trigger `formChange` when needed?